### PR TITLE
Test for move

### PR DESCRIPTION
**Pull request by yp1383**
Test Cases for move up and move down

### CODE REVIEW

File: creation/tests.py [hunk 1]
```diff
@@ -82,3 +82,33 @@ def test_creation_page_delete_dayvenue(self):
...
+            day=self.test_day, venue=self.test_venue, pos=1
+        )
+        self.test_dayvenue2 = DayVenue.objects.create(
+            day=self.test_day, venue=self.test_venue, pos=2
+        )
+
+        self.client.login(username=self.test_username, password=self.test_password)
+        self.client.get("/creation/2/edit/2/up")
+
+        self.assertFalse(self.test_dayvenue2 == 2)
```
Comment (sballan) [inline creation/tests.py:99]: This will always be false, because `test_dayvenue2` is not an integer

File: creation/tests.py [hunk 2]
```diff
@@ -82,3 +82,33 @@ def test_creation_page_delete_dayvenue(self):
...
+            day=self.test_day, venue=self.test_venue, pos=1
+        )
+        self.test_dayvenue2 = DayVenue.objects.create(
+            day=self.test_day, venue=self.test_venue, pos=2
+        )
+
+        self.client.login(username=self.test_username, password=self.test_password)
+        self.client.get("/creation/2/edit/1/down")
+
+        self.assertFalse(self.test_dayvenue1 == 1)
```
Comment (sballan) [inline creation/tests.py:114]: same here

**Review by sballan — CHANGES_REQUESTED** [review]
make sure you're testing the `pos`

File: creation/tests.py (hunk 1 above)
Comment (yp1383) [reply creation/tests.py:99]: yes, I just saw that lol. Working on it.

File: creation/tests.py (hunk 2 above)
Comment (yp1383) [reply creation/tests.py:114]: Same as above

File: creation/tests.py (hunk 1 above)
Comment (yp1383) [reply creation/tests.py]: Done

**Review by wangsnuxiao — APPROVED** [review]

**Pull request merged**